src/components/example/EDialog.tsx

```
@@ -0,0 +1,55 @@
+                    <div className="grid grid-cols-4 items-center gap-4">
+                        <Label htmlFor="username" className="text-right">
+                            Username
+                        </Label>
+                        <Input
+                            id="username"
+                            defaultValue="@peduarte"
+                            className="col-span-3"
+                        />
+                    </div>
+                </div>
+                <DialogFooter>
```

ESLint

[error] Delete `········`

(prettier/prettier)